lib/hexpm_web/views/error_view.ex

```diff
@@ -8,7 +8,8 @@ defmodule HexpmWeb.ErrorView do
       error: true,
       status: status,
       message: message(status),
-      container: "container error-view"
+      container: "container error-view",
+      current_user: assigns[:current_user]
```

Comment:
You will want to add this in the render on line 26 as well.

Reply:
Thanks, makes sense :)